src/solver/Main.java

```diff
@@ -1,7 +1,98 @@
 package solver;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
 public class Main {
+
+    static String getHelp() {
+        String helpMsg = "This program is intended to solve Linear Equations with any amount of variables using Gauss-Jordan Elimination. \n\n" +
+                "Options: " +
+                "\n\t * -in [pathToFile] reads file with the first number - number of equations and variables, and augmented matrix of linear equation" +
+                "\n\t * -out [pathToFile] writes results to the output file " +
+                "\n\t * -h shows help information " +
+                "\n\nExample: \njava Solver -in in.txt -out out.txt" +
+                "\n\nExample of in.txt:\n" +
+                "3\n" +
+                "1 1 2 9\n" +
+                "2 4 -3 1\n" +
+                "3 6 -5 0";
+        return helpMsg;
+    }
+
+    /**
+     * Program should start with necessary options -in [pathToFileWithMatrix] and -out [pathToOutputFile]
+     * Example: > java Solver -in in.txt -out out.txt
+     * @param args
+     */
     public static void main(String[] args) {
```

Comment:
Method main is very big. If your method does two or three different things at a time then you should consider splitting the functionality of this method into other methods (single responsibility principle) and just try to make your methods as small as possible.